<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCostsWindow</class>
 <widget class="QDialog" name="AddCostsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: white;
border: 2px solid rgb(13,110,253);
border-radius: 6px;
color:rgb(0, 0, 0);
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="addCSumLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: white;
border: 1px solid rgb(206,212,218);
border-radius: 6px;
color:rgb(0, 0, 0);
}
QLineEdit:focus {
border: 2px solid rgb(143,184,243);
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhLatinOnly</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>Введите сумму</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addCCategoryBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
background-color: white;
border: 1px solid rgb(206,212,218);
border-radius: 6px;
color:rgb(0, 0, 0);
}
QComboBox:focus {
border: 2px solid rgb(143,184,243);
}</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Выберите категорию</string>
   </property>
   <item>
    <property name="text">
     <string>Продукты</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Всё для дома</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Кафе</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Коммун. услуги</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Транспорт</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Здоровье</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Красота</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Одежда</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Досуг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Прочее</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addCRejectButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>255</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
border:1px solid rgb(108,117,125);
border-radius: 6px;
background-color: rgb(108,117,125);
}
QPushButton:focus {
border:3px solid rgb(92,99,106);
outline: none;
}
QPushButton:hover {
border-color: rgb(92,99,106);
background-color: rgb(92,99,106);
}
</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCAddButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>255</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
border:1px solid rgb(25, 135, 84);
border-radius: 6px;
background-color: rgb(25, 135, 84);
}
QPushButton:focus {
border: 3px solid rgb(21, 115, 71);
outline: none;
}
QPushButton:hover {
border-color: rgb(21, 115, 71);
background-color: rgb(21, 115, 71)
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="addCChooseDate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
background-color: white;
border: 1px solid rgb(206,212,218);
border-radius: 6px;
color:rgb(0, 0, 0);
}
QDateEdit:focus {
border: 2px solid rgb(143,184,243);
}
QCalendarWidget QToolButton {
color: white;
background-color: rgb(25, 135, 84);
}
QCalendarWidget QAbstractItemView:enabled {
selection-color: rgb(13, 110, 253);
selection-background-color: rgb(248, 249, 250);
}
QCalendarWidget QWidget#qt_calendar_navigationbar {
background-color:  rgb(25, 135, 84);
}</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2100</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy.MM.dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="addCComment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>340</width>
     <height>105</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
background-color: white;
border: 1px solid rgb(206,212,218);
border-radius: 6px;
color:rgb(0, 0, 0);
}
QTextEdit:focus {
border: 2px solid rgb(143,184,243);
}</string>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Trebuchet MS'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Напишите небольшой комментарий, если нужно</string>
   </property>
  </widget>
  <widget class="QFrame" name="addCAddButtonFrame">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>253</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(255,255,255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="addCRejectButtonFrame">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>252</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(255,255,255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>addCAddButtonFrame</zorder>
  <zorder>addCRejectButtonFrame</zorder>
  <zorder>addCSumLine</zorder>
  <zorder>addCCategoryBox</zorder>
  <zorder>addCRejectButton</zorder>
  <zorder>addCAddButton</zorder>
  <zorder>addCChooseDate</zorder>
  <zorder>addCComment</zorder>
 </widget>
 <tabstops>
  <tabstop>addCSumLine</tabstop>
  <tabstop>addCChooseDate</tabstop>
  <tabstop>addCCategoryBox</tabstop>
  <tabstop>addCComment</tabstop>
  <tabstop>addCAddButton</tabstop>
  <tabstop>addCRejectButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
